HttpFs.IntegrationTests/Tests.fs

```diff
@@ -434,4 +434,41 @@ let tests =
 
       for fileName in [ "file1.txt"; "file2.gif" ] do
         Expect.stringContains response fileName "response should contain filename"
+
+    ftestCase "multipart/mixed returns form values" <| fun _ ->
+      let firstCt, secondCt, thirdCt, fourthCt, fifthCt, fileContents =
+        ContentType.parse "text/plain" |> Option.get,
+        ContentType.parse "text/plain" |> Option.get,
+        ContentType.parse "application/json" |> Option.get,
+        ContentType.parse "application/atom+xml" |> Option.get,
+        ContentType.parse "application/x-doom" |> Option.get,
+        "Hello World"
+
+      let req =
+        Request.create Post (uriFor "/multipart")
+        |> Request.body (BodyForm
+          [ NameValue ("submit-name", "Larry")
+            MultipartMixed
+              ("files",
+                [
+                  "file1.txt", firstCt, Plain fileContents
+                  "file2.gif", secondCt, Plain fileContents
+                  "file3.json", thirdCt, Plain fileContents
+                  "file4.rss", fourthCt, Plain fileContents
+                  "file5.wad", fifthCt, Plain fileContents
```

Comment:
of cash?

Reply:
I'm not sure I understand?

Reply:
It was just a joke ;). A wad of cash. Bad joke.

Reply:
I actually went to go search the codebase for "cash" thinking I left a typo behind =)